apps/pdc-frontend/src/util/cspConfig.ts

```diff
@@ -1,98 +1,132 @@
 import { BLOB, DATA, EVAL, getCSP, INLINE, nonce, NONE, SELF, STRICT_DYNAMIC } from 'csp-header';
+import mergeWith from 'lodash.mergewith';
 import { createOpenFormsApiUrl } from '@/util/openFormsSettings';
 // Using "//*" in JavaScript, especially with VSCode, can disrupt syntax highlighting and code analysis, causing confusion and hindering development.
-const formatURL = (url: string): string => `https://${url}`;
+export const formatURL = (url: string): string => `https://${url}`;
 const getOpenFormsHost = () => {
   return createOpenFormsApiUrl()?.host || '';
 };
+type Object = Record<string, any>;
 
-const ogonePaymentServices = {
-  'form-action': [process.env.OGONE_PAYMENT_SERVICE_URL ?? ''],
+// eslint-disable-next-line no-unused-vars
+type MergeCustomizer = (objValue: Object, srcValue: Object) => any;
+
+export const mergeCustomizer: MergeCustomizer = (objValue, srcValue) => {
+  if (Array.isArray(objValue)) {
+    return objValue.concat(srcValue);
+  }
+  return undefined;
+};
```

Comment:
I used `mergeWith` instead of merge because, while `_.merge` combines objects recursively and overwrites values with the source values for properties with the same key (including nested objects), `mergeWith` allows for a customizer function to control how values are merged, providing more flexibility

```ts
const data1 = [
  {
    a: 1,
    b: 2,
  },
];
const data2 = [
  {
    a: 3,
    b: 4,
  },
];

const mergedData = _.merge(data1, data2);
console.log(mergedData); // output [ { a: 3, b: 4 } ]



```